ethereum/permissioning/src/main/java/org/hyperledger/besu/ethereum/permissioning/NodeSmartContractPermissioningController.java

```diff
@@ -61,58 +55,19 @@ private static Bytes hashSignature(final String signature) {
   private static final Bytes FALSE_RESPONSE =
```

Comment:
this is duplicated on the new V2 controller

Reply:
Not really, this is using the 32-bit encoded response. The V2 is using a simple bool response (as per solidity types). They look similar but are different :)